src/General-Rules-Tests/ReAssignmentInIfTrueRuleTest.class.st

```diff
@@ -0,0 +1,34 @@
+Class {
+	#name : 'ReAssignmentInIfTrueRuleTest',
+	#superclass : 'ReAbstractRuleTestCase',
+	#category : 'General-Rules-Tests-Migrated',
+	#package : 'General-Rules-Tests',
+	#tag : 'Migrated'
+}
+
+{ #category : 'tests' }
+ReAssignmentInIfTrueRuleTest >> testRule [
```

Comment:
Later I would also like a test with this code:

`testMethod | var | 1 == 1 ifTrue: [ 1 + 1. var := 1 ] ifFalse: [ var := 2 ]`